feat: offboarding operator (#248)
moondance-labs/tanssi-docs: node-operators/operators/offboarding/prerequisites.md

---
title: Prerequisites
description: Before offboarding your Tanssi node, ensure you have wallet access and sufficient ETH for gas. This guide outlines crucial prerequisites.
icon: octicons-arrow-down-right-24
template: main.html
---

# Prerequisites

## Introduction {: #introduction }

Operator offboarding is the formal process by which node operators safely and transparently exit the Tanssi protocol. It ensures network integrity, security, and stability by providing clear steps for operators who wish to cease participation.

Operators play a critical role in consensus and network operations. Abruptly shutting down a node without following the proper procedures can negatively impact operators, potentially resulting in slashing.

> This guide provides a complete walkthrough of the operator offboarding process, including:

Reviewer: No, It doesn't anymore :)
State only what is covered in this guide

> This specific guide provides a walkthrough of the **prerequisites** needed before starting the operator offboarding process. Subsequent guides will cover signaling your intent, removing the operator, and decommissioning your node.

Reviewer: ```suggestion
This guide outlines the **prerequisites** for offboarding, and subsequent guides will walk you through the process.
```

- Signaling your intent to exit the protocol
- Safely removing your operator from Tanssi's middleware
- Preventing resource waste during offboarding
- Decommissioning your node and exiting associated vaults

If you have questions during offboarding, the Tanssi team can support you on [Discord](https://discord.gg/Jm2KH8xT7J){target=\_blank}.

## Prerequisites {: #prerequisites}

Before starting the offboarding process, ensure you have the following:

- Access to the Ethereum (EVM) wallet that controls your operator account
- Sufficient ETH in your wallet to cover gas fees for transactions

### Interaction Methods

> --8<-- 'text/node-operators/operators/onboarding/run-an-operator/prerequisites.md'

Reviewer: Explain why I need to interact with smart contracts, before starting to explain different methods
